### DISCUSSION TITLE
How to register custom handlebars?

### DISCUSSION THREAD

**Discussion by WG-** [microsoft-search/pnp-modern-search#3065] [Q&A]
I do not understand the documentation w.r.t. registering custom handlebars. https://microsoft-search.github.io/pnp-modern-search/extensibility/handlebars_customizations/

Where exactly and how do I register custom handlebar? 

I tried using
```
Handlebars.registerHelper("examStatus", function(topScorer){
	if(topScorer > 90) {
      return "over 90%";}
  	else {
      return "under 90%";
    }
})
```

Which does not work, which I also read from the documentation. But where else can I register new helpers? I believe I see some .typescript code on the manuel page. However, I cannot add any .ts code to SharePoint since I am not an administrator. Is there another way?

![image](https://github.com/microsoft-search/pnp-modern-search/assets/1480116/fbc715b2-c3f1-4402-81ff-4c9c0cb4d158)

**Comment by wobba** [comment]
See https://microsoft-search.github.io/pnp-modern-search/extensibility/#register-your-extensibility-library-with-a-web-part, and I'll make a doc update to reference back to the introduction page.
